<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothWidget</class>
 <widget class="QWidget" name="BluetoothWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>ËìùÁâôËÆæÁΩÆ</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1" colspan="2">
           <widget class="QPushButton" name="btn_connect">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>ËøûÊé•üîó</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ËÆæÂ§áÔºö</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_state">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resource.qrc">:/icon/disconnect</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_devices">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_refresh">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icon/refresh</normaloff>:/icon/refresh</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_connected_device">
       <property name="title">
        <string>Â∑≤ËøûÊé•ËÆæÂ§á</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>ÂêçÁß∞Ôºö</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_addr">
          <property name="text">
           <string>Âú∞ÂùÄÔºö</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_recv_clear">
         <property name="text">
          <string>Ê∏ÖÁ©∫Êé•Êî∂</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Êó∂Èó¥Êà≥</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>HexÊòæÁ§∫</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QTextEdit" name="edit_recv">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="4">
        <widget class="QTextEdit" name="edit_send"/>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>HexÂèëÈÄÅ</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>ÂõûËΩ¶Êç¢Ë°å</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_send_clear">
         <property name="text">
          <string>Ê∏ÖÁ©∫ÂèëÈÄÅ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_send">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>ÂèëÈÄÅ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
